.github/workflows/_ruby-package.yml

```diff
@@ -83,6 +82,7 @@ jobs:
     if: ${{ contains(github.ref, 'refs/tags/v') && contains(github.ref, inputs.package) }}
     steps:
       - uses: actions/checkout@v3
+      - run: git config --global --add safe.directory "$GITHUB_WORKSPACE"
```

Comment:
I'm not sure this works if the issue is with the container in the following action